### ISSUE TITLE
Finish hotfix branch leaves the commit history 1|1

### ISSUE THREAD

**Issue by thetaylormcrae** [Shaggy13spe/gitflow4code#18]
When finishing a hotfix branch, the commit history leaves the develop branch one commit ahead and behind the master branch.
![image](https://user-images.githubusercontent.com/2520379/52498220-ce267f00-2b9d-11e9-9202-6b1c4beab501.png)
![image 1](https://user-images.githubusercontent.com/2520379/52498222-ce267f00-2b9d-11e9-813e-1135fb3652c9.png)
![image 2](https://user-images.githubusercontent.com/2520379/52498271-f0b89800-2b9d-11e9-9b51-df41703db113.png)

**Comment by Shaggy13spe** [comment]
Hmmm. Just curious, after finishing the hotfix, you manually pushed both master and develop? I've not seen this behavior in our production repos and we've been using the extension for about 3 years.

**Comment by thetaylormcrae** [comment]
That's correct. After running `gitflow: hotfix: finish`, it doesn't update the remote repository.
